docs/network-flow-visibility.md

```diff
@@ -367,7 +367,11 @@ git clone --depth 1 --branch main https://github.com/vmware-tanzu/antrea.git &&
 or
 svn export https://github.com/vmware-tanzu/antrea/trunk/build/yamls/elk-flow-collector/
 ```
-
+To configure the export interval as `flowExportInterval` in [Configuration](#configuration), 
```

Comment:
Heard from @antoninbas: we need a new line before and after a code block or a bulleted list for antrea.io to show the document correctly.

Reply:
that's correct, but we have a CI job for that now so I don't hav to point it out anymore :)